### PR TITLE
Newlines, named JSON-s and some renaming.

### PR DESCRIPTION
**Pull request by dkorolev**

### CODE REVIEW

**Comment by dkorolev** [comment]
@sompylasar : FYI.

File: net/api/impl/posix_client.h [hunk 1]
```diff
@@ -106,7 +106,7 @@ class HTTPClientPOSIX final {
     return true;
   }
 
-  const HTTPRedirectableReceivedMessage& GetMessage() const { return *message_.get(); }
+  const HTTPRedirectableRequestData& GetMessage() const { return *message_.get(); }
```
Comment (sompylasar) [inline net/api/impl/posix_client.h]: `GetRequestData`?

**Comment by sompylasar** [comment]
LGTM

File: net/api/impl/posix_client.h (hunk 1 above)
Comment (dkorolev) [inline net/api/impl/posix_client.h]: Good eye! Fixed in 93eb412, https://github.com/KnowSheet/Bricks/pull/50/.

**Pull request merged**